protocol/streaming/full_node_streaming_manager.go

```diff
@@ -98,6 +101,12 @@ func (sm *FullNodeStreamingManagerImpl) Enabled() bool {
 	return true
 }
 
+func (sm *FullNodeStreamingManagerImpl) SetWebsocketServer(
+	server *ws.WebsocketServer,
+) {
+	sm.websocketStreamServer = server
+}
+
```

Comment:
I would actually decouple this from the manager

we only need this to stop the server, right? we can just do it directly in app.go, no need to go through the manager